incubator/appdaemon/0.1.0/test_values.yaml

```diff
@@ -0,0 +1,52 @@
+image:
+  repository: ghcr.io/truecharts/appdaemon
+  pullPolicy: IfNotPresent
+  tag: "v4.0.8"
+
+strategy:
+  type: Recreate
+
+# Configure the Security Context for the Pod
+podSecurityContext:
+  runAsNonRoot: false
+  runAsUser: 0
+  runAsGroup: 0
+  fsGroup: 0
```

Comment:
This can run as non-root, it's designed for it by the k8s-at-home team.

Reply:
```suggestion
# Configure the Security Context for the Pod
podSecurityContext:
  runAsNonRoot: true
  runAsUser: 568
  runAsGroup: 568
  fsGroup: 568
```

Is this what you are expecting here?

Reply:
Yes.